src/Security/Items.cs

```diff
@@ -147,11 +147,12 @@ public static NSData QueryAsData (SecRecord query, bool wantPersistentReference,
 					if (max == 1)
 						return new NSData [] { new NSData (ptr, false) };
 
-					var array = new NSArray (ptr);
-					var records = new NSData [array.Count];
-					for (uint i = 0; i < records.Length; i++)
-						records [i] = new NSData (array.ValueAt (i), false);
-					return records;
+					using (var array = new NSArray (ptr)) {
```

Comment:
Any reason you used using blocks here but statements up above?

Reply:
No reason at all. Switched to using statement.